config/filter.d/apache-common.conf

```diff
@@ -5,10 +5,16 @@
 
 # Load customizations if any available
 after = apache-common.local
+# Uncomment the following line if you use ErrorLog = syslog
+before = common.conf
```

Comment:
So an advice in the .conf file ?
I thought about doing so, and finally created the `.local` file...